pkg/webhook/nginxingress.go

```diff
@@ -209,41 +207,20 @@ func (n *nginxIngressResourceValidator) Handle(ctx context.Context, req admissio
 			return admission.Allowed("")
 		}
 
-		lgr.Info("checking if IngressClass already exists")
-		ic := &netv1.IngressClass{
-			ObjectMeta: metav1.ObjectMeta{
-				Name: nginxIngressController.Spec.IngressClassName,
-			},
-		}
-		lgr.Info("attempting to get IngressClass " + ic.Name)
-		err = n.client.Get(ctx, client.ObjectKeyFromObject(ic), ic)
-		if err == nil {
-			lgr.Info("denied because IngressClass already exists")
-			return admission.Denied(fmt.Sprintf("IngressClass %s already exists. Delete or use a different spec.IngressClassName field", ic.Name))
-		}
-		// err can still be populated as not found after this so be careful to overwrite for accurate metrics if needed
-		if !k8serrors.IsNotFound(err) {
-			lgr.Error(err, "get IngressClass")
-			return admission.Errored(http.StatusInternalServerError, fmt.Errorf("getting IngressClass %s: %w", ic.Name, err))
+		lgr.Info("checking if it collides")
```

Comment:
nit: can you change this to checking if ingresscontroller collides/just get rid of the usage of "it"? feel like more context is always better for these

Reply:
will do in the next pr